estimation.py

```diff
@@ -67,6 +67,7 @@ def estimate_memory(job_config: JobConfig):
         pp=job_config.experimental.pipeline_parallel_degree,
         world_size=world_size,
         enable_loss_parallel=job_config.training.enable_loss_parallel,
+        dp_type=job_config.experimental.data_parallel_type,
```

Comment:
I'm not sure what's the best category here. I think FSDP is not experimental. Is DDP without compiled_autograd also experimental? I'm OK with leaving this option as experimental if things are not clear.